cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -40,14 +40,14 @@ To use the **Docker image**:
 
 - Once Docker has started, pull down the *CDAP Docker Image* from the Docker hub using::
 
-    $ docker pull caskdata/cdap-standalone
+    $ docker pull caskdata/cdap-standalone:\ |literal-release|
```

Comment:
If you are worried about this line, GitHub is not Sphinx, and knows nothing about the substitution ```|literal-release|```. It will work fine after processing in Sphinx, and in fact that's the only way you can check this.